docs/csharp/misc/cs0077.md

```diff
@@ -39,7 +41,11 @@ class M
       o1 = new C();  
       o2 = new S();  
   
-      s = o2 as S;  
+      // Use pattern matching instead of as
+      if (o2 is S sValue)
+      {
+          s = sValue;
+      } 
       // CS0077, S is not a reference type.  
       // Try the following line instead.  
       // c = o1 as C;  
```

Comment:
✂️ These are no longer needed at all, because we now present other solution (with the `is` operator).

Reply:
Resolved with new commit.

Reply:
I'm not sure if the commit you mean you already pushed, because these comments are still there.
Let's remove all comments from lines 51 to 53 and we'll be good 👍